src/process.cr

```diff
@@ -129,19 +129,26 @@ class Process
     pid
   end
 
-  # The standard `IO` configuration of a process:
-  #
-  # * `nil`: use a pipe
-  # * `false`: no `IO` (`/dev/null`)
-  # * `true`: inherit from parent
-  # * `IO`: use the given `IO`
-  alias Stdio = Nil | Bool | IO
+  # How to redirect the standard input, output and error IO of a process.
+  enum Redirect
+    PIPE
+    CLOSE
+    INHERIT
+  end
+
+  PIPE    = Redirect::PIPE
```

Comment:
As per #3084 the aliases are to be used *outside* of the `Process` class, so we don't have to type the lengthy `Process::Redirect::CLOSE` but `Process::CLOSE`. I wish symbols were transformed automatically to enum values (when an enum is expected), so we could just set `:close`.